### PR TITLE
Add action button to Employee, Add authentication to the action button, Fixed Role add, Connected Role delete -Stephan

### PR DESCRIPTION
**Pull request by thedirtyham**
Created the action button and linked it to adding new roles to an employee, created authentication to use the action button. You now must be logged in to use the action button. Fixed the endpoint in roles/put that was causing an error for adding roles. Connected the Delete role function to the correct button and current API endpoint call. 

**TEST PLAN**
Make sure you can see the action button on the employee's page by logging in. Make sure you ca add a new role to an existing employee. 
Add a new role
Attach new group to role
Delete new role

### CODE REVIEW

**Review by eliasargandara — COMMENTED** [review]
On the Role page, when you select the Action button and then choose "Add Role" and then "Delete Role" it displays two different selection areas. It should close the previous one and then display the other one.

**Comment by thedirtyham** [comment]
On the roles page when you click on the action button and then on the delete button the action  button disappears. No other section appears. Low key confused what you are saying.

**Comment by eliasargandara** [comment]
Sorry, I must have been tired. I meant the Employee page. Here's a picture of what I was talking about. There's a selection menu for both actions at the same time. It should only be possible to have one open at a time.
employees_page_screenshot

**Comment by thedirtyham** [comment]
the button that goes away is in the roles page, my button does not go away when you click the remove roles. I merged sals code and pushed the combine changes.

File: dashboard/static/js/Roles.js [hunk 1]
```diff
@@ -69,6 +70,49 @@ $(document).ready(function(){
         });
     });
         
+<<<<<<< HEAD
```
Comment (eliasargandara) [inline dashboard/static/js/Roles.js]: There shouldn't be merge conflicts showing up in the code.

**Pull request merged**